### ISSUE TITLE
Error "index out of bounds" in home page

### ISSUE THREAD

**Issue by nojhan** [IOHprofiler/IOHanalyzer#49]
After having pulled version 5c19745 (from Tue Mar 10 12:59:37 2020 +0100), I get the following error while accessing the root page:
```R
Loading required package: shiny
Listening on http://127.0.0.1:3229
Warning: Error in [[: indice hors limites
  46: updateSelectInput
  45: <observer> [server/upload.R#8]
   2: shiny::runApp
   1: runServer
```
I guess « indices hors limites » would translate "index out of bounds".

I have no idea how to debug in R, so I would welcome any doc in the README about that :-)

**Comment by nojhan** [comment]
The same problem occurs with version 5b68ab4 (InDevelopment branch).

**Comment by Dvermetten** [comment]
Thanks for reporting this issue.
This is related to the repository-detection: basically, the server assumes that a folder called 'repository' exists in the users home-directory, which it then uses to look for rds-files (R's internal format) to ease with loading datasets (avoids us having to unzip the same data every time). 
However, it seems like the error-handling when this folder is not present is lacking, causing the issue you describe above. I will fix this soon and update the documentation to explain how to use such a repository-folder.

**Comment by nojhan** [comment]
Indeed, after creating a `repository` directory in home, it loads correctly. Thanks.

**Comment by Dvermetten** [comment]
This has been resolved and added to PR #48, which has now been merged to master.
